geoalchemy2/functions.py

```diff
@@ -84,6 +84,8 @@ class ST_TransScale(GenericFunction):
             type = Geometry
     """
 
+    # Set _register to False in order to not register this class in
+    # sqlalchemy.sql.functions._registry. Only its childs will be registered.
```

Comment:
s/to not/not to/
s/childs/children/

Thanks

Reply:
I  can do it myself tomorrow.

Reply:
Thanks, I fixed it :)